meta-cube/recipes-core/lxc/files/overlayrestore

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+restore()
```

Comment:
This routine should have the inputs/arguments commented, and also an overall comment about what it does.